uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -629,6 +629,16 @@
         });
 
         describe('save leave request', function () {
+          describe('allows multiple save', function() {
+            beforeEach(function() {
+              $ctrl.submit();
+            });
+
+            it('does not allow user to submit again', function () {
+              expect($ctrl.didSubmit).toBeTruthy();
```

Comment:
even though the button is not visible its still disabled
ng-disabled="!$ctrl.canSubmit() || $ctrl.didSubmit" from html